src/acl/RegexData.cc

```diff
@@ -148,12 +141,14 @@ compileRE(std::list<RegexPattern> &curlist, const SBufList &RE, int flags)
  * called only once per ACL.
  */
 static int
-compileOptimisedREs(std::list<RegexPattern> &curlist, const SBufList &sl)
+compileOptimisedREs(std::list<RegexPattern> &curlist, const SBufList &sl, const bool caseInsensitiveDefault)
```

Comment:
Just to clarify the meaning of this boolean parameter:

```suggestion
compileOptimisedREs(std::list<RegexPattern> &curlist, const SBufList &sl, const bool startCaseInsensitively)
```